wannierberri/__system.py

```diff
@@ -409,6 +409,66 @@ def check_hermitian(self,XX):
         else:
             print (f"{XX} is missing,nothing to check")
 
+    def set_structure(self, positions, atom_numbers, magnetic_moments=None):
+        """
+        Set atomic structure of the system.
+
+        Parameters
+        ----------
+        positions : (num_atom, 3) array_like of float
+            Atomic positions in fractional coordinates.
+        atom_numbers: (num_atom,) list of int
```

Comment:
would be good to provide `atom_labels` -  a list of any labels (string,number, or in general any hashable type, e.g.`['Fe','O1','O2','O1',2,3.14]` and then internally map them to numbers to pass to the `spglib`